spec/truck_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+describe Truck do
+  describe 'validations' do
+    subject { truck.valid? }
+
+    let(:truck) { Truck.new(license_plate: license_plate, capacity: capacity) }
+    let(:license_plate) { '1234' }
+    let(:capacity) { 100 }
+
```

Comment:
lets also add a basic spec that asserts its valid so that if we ever add more validations it would highlight when its valid or invalid

`it { is_expected.to be_true }`